 <launch>



  <!-- <node name="local_costmap" pkg="costmap_2d" type="costmap_2d_node" output="screen"> -->
    <!-- <param name="local_costmap" type="string" value="local_costmap"/> -->
  <!-- </node> -->
  <!-- <node name="global_costmap" pkg="costmap_2d" type="costmap_2d_node" output="screen"> -->
    <!-- <param name="global_costmap" type="string" value="global_costmap"/> -->
  <!-- </node> -->


  <node name="decomp_node" pkg="mpc_planner_standalone" type="decomp_node" output="screen">
    <!-- <rosparam file="$(find jackal_navigation)/params/base_local_planner_params.yaml" command="load" /> -->
    <!-- <rosparam file="$(find jackal_navigation)/params/move_base_params.yaml" command="load" /> -->
    
    <!-- <rosparam file="$(find mpc_planner_standalone)/config/costmap_common_params.yaml" command="load" ns="global_costmap" /> -->
    <!-- <rosparam file="$(find jackal_navigation)/params/odom_nav_params/global_costmap_params.yaml" command="load" /> -->
    
    <rosparam file="$(find mpc_planner_standalone)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find mpc_planner_standalone)/config/local_costmap_params.yaml" command="load"/>
  </node>
  
</launch>